server.js

```diff
@@ -22,170 +22,25 @@
-    // The REDIRECT_URI must be added to the External Application settings in Banno People.
-    // *NOTE* it is case sensitive.
-    // See more details at https://jackhenry.dev/open-api-docs/plugins/architecture/ExternalApplications/
-    const REDIRECT_URI = `http://localhost:${config.app_port}/dynamic`
-
-    let state
-    let codeVerifier
-    let codeChallenge
-    if (!req.query.code || !req.query.state) {
-        // If we are in this state, then we are starting a new authorization flow.
-        if (req.query.state) {
-            stateStore.delete(req.query.state)
-        }
-
-        // PKCE (Proof Key for Code Exchange) is an OAuth extension that adds additional security to the Authorization Code flow.
-        // Here we create both the Code Verifier and Code Challenge.
-        // See more details at https://tools.ietf.org/html/rfc7636
-        codeVerifier = createCodeVerifier(codeVerifier)
-        codeChallenge = createCodeChallenge(codeVerifier)
-
-        // We save the Code Verifier for later use in the authorization flow.
-        state = createState()
-        stateStore.set(state, codeVerifier)
-        res.cookie(`STATE_${state}`, codeVerifier, { httpOnly: true, sameSite: 'lax' })
-
-        // Build up the authorization URL piece by piece, starting with the authorization endpoint.
-        const authBaseURL = `${config.api.environment}/a/consumer/api/v0/oidc/auth`
-
-        // See more about scopes (and claims) at https://jackhenry.dev/open-api-docs/authentication-framework/overview/OpenIDConnectOAuth/
-        const scopesParameterEncoded = `?scope=${encodeURIComponent('openid profile https://api.banno.com/consumer/auth/accounts.readonly')}`
-
-        const responseTypeParameter = `&response_type=code`
-
-        const clientIdParameter = `&client_id=${config.api.client_id}`
-
-        const redirectUriParameterEncoded = `&redirect_uri=${encodeURIComponent(REDIRECT_URI)}`
-
-        const stateParameter = `&state=${state}`
-
-        // Adapted from https://jackhenry.dev/open-api-docs/authentication-framework/overview/OpenIDConnectOAuth/.
-        //
-        // Here we request a Publicly Available Claim.
-        //
-        // If we had instead requested a Restricted Claim, the administrator at the financial institution
-        // would also need to enable that restricted claim for the External Application used by this example
-        // in order to actually receive data for that claim.
-        const claims = {
-            'https://api.banno.com/consumer/claim/institution_id': null,
-            // If you uncomment this line for the 'Unique Customer Identifer' Restricted claim,
-            // the administrator at the financial institution would also need to enable that restricted claim
-            // for the External Application used by this example in order to actually receive data for that claim.
-            //'https://api.banno.com/consumer/claim/customer_identifier': null,
```

Comment:
A super common thing for folks to do is to try to get the `https://api.banno.com/consumer/claim/customer_identifier`, and we've lost that ease of 'uncomment this line and the corresponding console.log line' aspect.